Jenkinsfile_CNP

```diff
@@ -46,11 +46,11 @@ withPipeline(type, product, component) {
 
     after('checkout') {sh 'yarn cache clean'}
 
-  before('buildinfra:prod') {
-    currentBuild.result = 'SUCCESS'
-    currentBuild.description = "SUCCESS"
-    error 'XUI  projects are not authorised to be in production environments yet.'
-  }
+//  before('buildinfra:prod') {
```

Comment:
remove or add comment why has been commented out

Reply:
that stops code to deploy to prod, in any case if we want block prod deployment in future we can uncomment that